packages/vscode-extension/src/commandHandlers/params.ts

```diff
@@ -44,6 +44,8 @@ export function validateParamInput(
   return undefined;
 }
 
+const DEBOUNCE_TIME = 200;
```

Comment:
There's no need for this constant, it's only used in that timeout. Also it's not a debounce time anymore